configure.py

```diff
@@ -1030,6 +1031,7 @@ def find_headers(repodir, excluded_dirs):
                 'service/broadcast_tables/experimental/lang.cc',
                 'tasks/task_manager.cc',
                 'rust/wasmtime_bindings/src/lib.rs',
+                'utils/s3/client.cc',
```

Comment:
Please put it next to the other utils.

Reply:
"Other utils" are somewhat dispersed throughout this list already :confused:

Reply:
Well, next to some other group, let's not increase the dispersion.